app/src/main/java/com/testtask/apptesttask/model/repository/CharactersRepository.kt

```diff
@@ -14,31 +14,55 @@ class CharactersRepository @Inject constructor(
     private val marvelService: MarvelService,
     private val appConfig: AppConfig,
     private val schedulers: SchedulersProvider,
-    private val sharedPreference: Prefs
+    private val prefs: Prefs
 ) {
 
-    private fun getApiCharacters(): Single<List<ApiCharacter>> = marvelService
+    private fun getApiCharacters() = marvelService
```

Comment:
Remove this method and use `marvelService.getCharacters` in `Single.zip` with `characterDataWrapper` in `BiFunction`.